notebooks/06_spatial_autocorrelation.md

```diff
@@ -452,6 +450,13 @@ Similarly, inference can also be carried out by relying on computational simulat
 
 
 
+##  Further Reading
+
+- {cite}`anselin1989special`. "What is special about spatial data? Alternative perspectives on spatial data analysis." UC Santa Barbara: National Center for Geographic Information and Analysis.
```

Comment:
A little narrative of like, what each of these sources provides would be useful!